x/valset-pref/validator_set.go

```diff
@@ -195,7 +193,6 @@ func (k Keeper) UndelegateFromValidatorSet(ctx sdk.Context, delegatorAddr string
 
 	// Step 6
```

Comment:
Can we give a synopsis of Step 6 and Step 7 in the comments like the others? Was nice/helpful

Reply:
I mentioned this on slack, but I personally don't understand if this part of the algorithm is even reachable, thus should be deleted. Will hold this comment for now and either add more comment or delete this once I gain context